From b6271dfad845b9fcaf36ee6d2d19422aada09a1a Mon Sep 17 00:00:00 2001
From: Valentin Caron <valentin.caron@foss.st.com>
Date: Mon, 23 May 2022 15:15:19 +0200
Subject: [PATCH] arm64: dts: st: add RTC on stm32mp25x

Add compatible, clock, and interrupt properties of STM32 RTC on
stm32mp25x SOCs.

Signed-off-by: Valentin Caron <valentin.caron@foss.st.com>
Change-Id: Ib698d57d6530a3ddd1de43cd6bc85c1f3bfb0b12
---
 arch/arm64/boot/dts/st/stm32mp251.dtsi | 10 ++++++++++
 1 file changed, 10 insertions(+)

diff --git a/arch/arm64/boot/dts/st/stm32mp251.dtsi b/arch/arm64/boot/dts/st/stm32mp251.dtsi
index 820cfc753b0f..cc0c2aee6f89 100644
--- a/arch/arm64/boot/dts/st/stm32mp251.dtsi
+++ b/arch/arm64/boot/dts/st/stm32mp251.dtsi
@@ -641,6 +641,16 @@ gpiok: gpio@442e0000 {
 			};
 		};
 
+		rtc: rtc@46000000 {
+			compatible = "st,stm32mp25-rtc";
+			reg = <0x46000000 0x400>;
+			clocks = <&scmi_clk CK_SCMI_RTC>,
+				 <&scmi_clk CK_SCMI_RTCCK>;
+			clock-names = "pclk", "rtc_ck";
+			interrupts-extended = <&exti2 17 IRQ_TYPE_LEVEL_HIGH>;
+			status = "disabled";
+		};
+
 		pinctrl_z: pinctrl@46200000 {
 			#address-cells = <1>;
 			#size-cells = <1>;
-- 
2.39.5

